backend/src/schema/resolvers/searches.js

```diff
@@ -8,7 +8,8 @@ export default {
       // see http://lucene.apache.org/core/8_3_1/queryparser/org/apache/lucene/queryparser/classic/package-summary.html#package.description
       const myQuery = query
         .replace(/\s+/g, ' ')
-        .replace(/[[@#:*~\\$|^\]?/"'(){}+?!,.-;]/g, '')
+        .replace(/[[@#:*~\\$|^\]?/"'(){}+?!,.;]/g, '')
+        .replace(/-/g, ' ')
```

Comment:
This, as well, is the job of the tokenizer. Maybe you could search for "JavaScript tokenizer" to find a less hacky solution. Or look for a solution that is related to elastic search: "elastic search normalize query". 

You could add tests for this behaviour as well, at least for documentation purposes.